staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po

```diff
@@ -100,16 +100,16 @@ msgid ""
 "\t\tkubectl top pod -l name=myLabel"
 msgstr ""
 "\n"
-"\t\t# 显示 default namespace 下所有 pods 下的 metrics\n"
+"# 显示默认命名空间中所有 Pod 的指标\n"
```

Comment:
I notice that some of these have tabs (`\t\t`) removed.  Was that intentional?

Reply:
Sorry, this is my mistake, I will correct it.